shared/middleware/net_sim_api.rb

```diff
@@ -45,6 +45,16 @@ def get_table(shard_id, table_name)
     get_table(shard_id, table_name).to_a.to_json
   end
 
+  # GET /v3/netsim/<shard-id>/<table-name>@<min_id>
+  #
+  # Returns all of the rows in the table with id >= min_id
+  #
+  get %r{/v3/netsim/([^/]+)/(\w+)@([0-9]+)$} do |shard_id, table_name, min_id|
```

Comment:
There is a test for this new route, it starts on line 53 of test_net_sim_api.rb.